docs/integrations/editors.md

```diff
@@ -160,6 +160,16 @@ If you need to do anything special to make your virtualenv work and install _Bla
 example you want to run a version from main), create a virtualenv manually and point
 `g:black_virtualenv` to it. The plugin will use it.
 
+If you would prefer to use the system installation of _Black_ rather than a virtualenv,
+then add this to your vimrc:
+
+```
+let g:black_use_system = 1
+```
+
+Note that when the system installation is in use, the `:BlackUpgrade` command is not
+usable.
```

Comment:
Is it hard to make it work?

Reply:
It would be impractical (or impossible) for the vim plugin to upgrade the system installation of `black. Maybe that's not what you're asking about, though.

Are you asking whether the vim plugin could still upgrade the virtualenv installation even when `g:black_use_system` is true? I think yes--it would be easy to redo the logic (untested):
```
  if vim.eval("g:black_use_system ? 'true' : 'false'") == "true" and not upgrade:
     # Nothing needed to be done.
      return True
```

...but I don't think that would be very good from a user standpoint. If a user has set `g:black_use_system`, then they have indicated that that do not want to use a virtualenv, so installing a virtualenv (or upgrading an existing one) would be pointless--the virtualenv won't be used anyway.

At worst, if a user has `g:black_use_system` enabled by error and _does_ want to use a virtualenv, then when installing/upgrading the virtualenv has no effect on the `black` version in use, that would make it appear as if the virtualenv installation/upgrade has done something wrong.


This is all from my perspective, at least--I just think it would be least surprising to have `:BlackUpgrade` visibly fail rather than do something that is unwanted or have the results be masked. If I'm wrong on that, let me know and I can certainly change the PR.

Reply:
Yeah I meant upgrading the system installation. Fair enough!